CHANGELOG.md

```diff
@@ -40,6 +40,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-2853**
+  - Move OAUTH_PROVIDER to lambda env variables to address regression in 9.9.2/CUMULUS-2275
```

Comment:
I don't think CUMULUS-2275 is the right ticket number? Should it be 2853?

Reply:
@markdboyd  the regression was from the changes introduced in CUMULUS-2275.    The ticket in the header *is* 2853.

Reply:
And..... possibly 2275 as well on this baseline.  I'll check the ticket ID on the main branch.

Reply:
Are you sure? Ticket 2275 seems to be related to CMA 1.3.0 changes?

https://bugs.earthdata.nasa.gov/browse/CUMULUS-2275

Reply:
2781 is the correct ticket 😓.    Updated: c6d5d35ff